modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/management_bar/components/FiltersDropdown.js

```diff
@@ -25,17 +25,20 @@ function FiltersDropdown() {
 	const [active, setActive] = useState(false);
 	const [query, setQuery] = useState('');
 
-	const [visibleFilters] = useState(
-		filtersState.filters.filter((filter) => !filter.invisible)
+	const visibleFilters = useMemo(
+		() => filtersState.filters.filter((filter) => !filter.invisible),
```

Comment:
Can haz more instances of the string "filter" on this line plz? 😜

Reply:
I'm trying my best to win a secret contest :P